So this is failing due to a formatting validation:

```
2019-03-13T14:06:23.3106300Z [ERROR] Failed to execute goal net.revelc.code.formatter:formatter-maven-plugin:2.8.1:validate (default) on project quarkus-maven-plugin: File '/home/vsts/work/1/s/devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java' has not been previously formatted.  Please format file and commit before running validation! -> [Help 1]
```

I have the eclipse formatter plugin installed, not sure why the code was not formatted correctly. I just pushed a reformat update.

--- a/devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java
+++ b/devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java
@@ -152,8 +152,8 @@ public class DevMojo extends AbstractMojo {
         }
         if (!found) {
             getLog().warn("The quarkus-maven-plugin build goal was not configured for this project, " +
-                                  "skipping quarkus:dev as this is assumed to be a support library. If you want to run quarkus dev" +
-                                  " on this project make sure the quarkus-maven-plugin is configured with a build goal.");
+                    "skipping quarkus:dev as this is assumed to be a support library. If you want to run quarkus dev" +
+                    " on this project make sure the quarkus-maven-plugin is configured with a build goal.");
             return;
         }
 
@@ -173,7 +173,7 @@ public class DevMojo extends AbstractMojo {
             if (debug == null) {
                 // debug mode not specified
                 // make sure 5005 is not used, we don't want to just fail if something else is using it
-                try (Socket socket = new Socket(InetAddress.getByAddress(new byte[]{127, 0, 0, 1}), 5005)) {
+                try (Socket socket = new Socket(InetAddress.getByAddress(new byte[] { 127, 0, 0, 1 }), 5005)) {
                     getLog().error("Port 5005 in use, not starting in debug mode");
                 } catch (IOException e) {
                     args.add("-Xdebug");
@@ -253,7 +253,7 @@ public class DevMojo extends AbstractMojo {
                 path = new File(jarPath);
             } else if (classFile.getProtocol().equals("file")) {
                 String filePath = classFile.getPath().substring(0,
-                                                                classFile.getPath().lastIndexOf(DevModeMain.class.getName().replace('.', '/')));
+                        classFile.getPath().lastIndexOf(DevModeMain.class.getName().replace('.', '/')));
                 path = new File(filePath);
             } else {
                 throw new MojoFailureException("Unsupported DevModeMain artifact URL:" + classFile);

--- a/devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java
+++ b/devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java
@@ -34,10 +34,6 @@ import java.util.jar.Manifest;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
 
-import io.quarkus.dev.ClassLoaderCompiler;
-import io.quarkus.dev.DevModeMain;
-import io.quarkus.maven.components.MavenVersionEnforcer;
-import io.quarkus.maven.utilities.MojoUtils;
 import org.apache.maven.artifact.Artifact;
 import org.apache.maven.execution.MavenSession;
 import org.apache.maven.model.Plugin;
@@ -54,6 +50,11 @@ import org.apache.maven.plugins.annotations.ResolutionScope;
 import org.apache.maven.project.MavenProject;
 import org.apache.maven.toolchain.Toolchain;
 import org.apache.maven.toolchain.ToolchainManager;
+
+import io.quarkus.dev.ClassLoaderCompiler;
+import io.quarkus.dev.DevModeMain;
+import io.quarkus.maven.components.MavenVersionEnforcer;
+import io.quarkus.maven.utilities.MojoUtils;
 
 /**
  * The dev mojo, that runs a quarkus app in a forked process

